src/Dfe.PlanTech.Domain/Questionnaire/Models/Params.cs

```diff
@@ -6,5 +6,6 @@ public class Params
 
         public string SectionId { get; set; } = null!;
 
+        public string SectionSlug { get; set; } = null!;
```

Comment:
What is the purpose of this? Is this not retrievable from the slug?

Reply:
There are places where this is not accessible so I need this.

Reply:
It's never actually used anywhere though. It's passed around a ton through various views, but isn't actually used by the vast majority of them.

The only place I can find where `SectionSlug` is really necessary but also missing, is the `SubmitAnswer` route, which could easily be fixed by adding it into the route matcher, then as a parameter in the action signature.

Really the only purpose to it that I can see is to ensure that the section slug is in the URL, which would be much cleaner done by the above as well. 

The slug should always be accessible, if it's not then it should be added, and this is much simpler, cleaner, and less buggy, than the current functionality.